src/ducks/Studio/AdvancedView/Histogram/index.js

```diff
@@ -72,10 +82,12 @@ const Histogram = ({ title, slug, ticker, date, datesRange, hasSort }) => {
     setDates(newDates)
   }
 
+  let isBucketAfterCurrentPrice = false
+
   return (
     <div className={styles.wrapper}>
       <h2 className={styles.title}>
-        Price movement
+        Price distribution
```

Comment:
Spent coin cost

Reply:
Addressed issue was fixed in the new commits